sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,907 @@
+capped by a limit in the `.sbtc` contract.  The `.sbtc` contract will refuse to
+admit a deposit if it would exceed this limit.  This limit can be increased (but
+not decreased) by Stacker vote, and it would start at a default value of 100 BTC.
+
+The reason for the limit is to allow the system to be stress-tested while
+minimizing risk to users in the early days.  Once Stackers are comfortable with
+maintaining the wallet and running signer daemons to manage the sBTC wallet,
+they can increase this limit.  They would remove it by simply setting the limit
+to 21 million BTC.
+
+Importantly, the capital limit is _not_ a function of the STX price.  This is
+because the BTC is safe if at least 30% of the stackers are honest -- they can 
+deny 69.999...% of dishonest Stackers the ability to abscond with the BTC.  Thus, this
+proposal calls for enabling Stackers to set the capital limit themselves, as
+long as the limit has at least 70% vote support.  To help build user trust
+in the safety of the BTC,  we expect Stackers to disclose additional information
+about themselves (e.g. business name, mailing address, corporate charter,
+portion of STX owned, etc.) in order to provide convincing evidence that this
+30% honest coalition exists.  If this 30% honest coalition exist, then there is
+no ROI for dishonest Stackers to participate if their goal is to steal the BTC.
+
+## Future Work: Recovery Mode
+
+This SIP proposes implementing the aforementioned Normal mode for sBTC.  A
+follow-up SIP may propose a Recovery mode, whereby instead of burning BTC while
+the system is frozen, the BTC is instead repurposed to directly fulfill pending
+withdraw requests.  While withdraw fulfillment will be considerably slower, at
+least the sBTC users will eventually be made whole in the event of a
+catastrophe.
+
+In Recovery mode, sBTC deposits would cease and all pending sBTC deposits would be
+cancelled.  Withdraws would be fulfilled in first-in first-out order from
+redirected PoX payouts.  Once a withdraw request has been
+satisfied -- i.e. the requester's BTC address has
+received _at least_ as many satoshis as reqeusted from the redirected PoX
+payouts -- then the withdraw request completes and the next requester's withdraw
+is processed.
+
+Once the Stacks blockchain supports Recovery Mode, the system would continue
+operating even if the system was frozen for many reward cycles.  The blockchain
+would not halt; instead, it would continue its operation to fulfill all withdraw
+requests while the BTC is seemingly inaccessible to Stackers.  PoX payouts and
+STX unlocks would only resume once the current set of Stackers control
+enough BTC to back each sBTC token, and are able to fulfill pending sBTC
+withdraws.
+
+The system would transition back to Normal mode once all pending withdraw
+transactions have been fulfilled, and once the current set of Stackers possess
+enough BTC to cover all outstanding sBTC tokens.
+
+# Related Work
```

Comment:
Should add a section about [Nomic](https://gist.github.com/mappum/da11e37f4e90891642a52621594d03f6), which is the most similar related work imo.

Reply:
cc @andreserrano